lab_1/main.py

```diff
@@ -5,39 +5,39 @@
 
 
 def tokenize(text: str) -> list:
-    """
-    Splits sentences into tokens, converts the tokens into lowercase, removes punctuation
-    :param text: the initial text
-    :return: a list of lowercased tokens without punctuation
-    e.g. text = 'The weather is sunny, the man is happy.'
-    --> ['the', 'weather', 'is', 'sunny', 'the', 'man', 'is', 'happy']
```

Comment:
change back function description (everywhere)

def remove_stop_words(tokens: list, stop_words: list) -> list:
     """
     Removes stop words
     :param tokens: a list of tokens
     :param stop_words: a list of stop words
     :return: a list of tokens without stop words
     e.g. tokens = ['the', 'weather', 'is', 'sunny', 'the', 'man', 'is', 'happy']
     stop_words = ['the', 'is']
     --> ['weather', 'sunny', 'man', 'happy']
     """
    YOUR CODE